test/functional/core/job_spec.lua

```diff
@@ -775,22 +840,36 @@ describe("pty process teardown", function()
   end)
 
   it("does not prevent/delay exit. #4798 #4900", function()
```

Comment:
Could use nvim_get_proc_children instead of a screen test.

Reply:
For the PIDs? It won't work with the child process launched with start (cmd.exe). Have to rethink this test for Windows.

Reply:
Not sure what you mean, but if `!start` is detached from nvim somehow, such that it wouldn't prevent nvim from exiting (if there were a nvim bug in its jobs teardown routine), then that defeats the purpose of the test.

Reply:
nvim will exit completely and `!start` terminates after running the program (ie. ping) but the program that `start` does run (ie ping) will not necessarily terminate if nvim terminates.

Reply:
> but the program that start does run (ie ping) will not necessarily terminate if nvim terminates.

That defeats the purpose of the test :)